### PR TITLE
fix(editor): Make sure deactivated status shows even for nodes with long names on new canvas (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary

Screenshot 2024-10-07 at 16 21 45

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/N8N-7797/cannot-see-full-node-name-here-and-deactivated-its-important-to-show

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/canvas/elements/nodes/render-types/CanvasNodeDefault.vue [hunk 1]
```diff
@@ -265,7 +267,8 @@ function openContextMenu(event: MouseEvent) {
 	align-items: center;
 }
 
-.label {
+.label,
+.disabledLabel {
 	font-size: var(--font-size-m);
 	text-align: center;
 	text-overflow: ellipsis;
```
Comment (mutdmour) [inline packages/editor-ui/src/components/canvas/elements/nodes/render-types/CanvasNodeDefault.vue:274]: can we remove this? We need to show full long names on canvas

**Review by mutdmour — APPROVED** [review]
not happy about shortening the names, will be super annoying. but thanks for fixing the bug

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Comment by janober** [comment]
Got released with [`n8n@1.63.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.63.0)

**Pull request merged**